### PR TITLE
Create LIT6624Maqabis.xml

### PR DESCRIPTION
**Pull request by thea-m**
I found no further references on this text. It would probably deserve some more attention.

### CODE REVIEW

File: new/LIT6624Maqabis.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        <revisionDesc>
+            <change who="DR" when="2022-05-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6624Maqabis" passive="BLorient770"/>
+                    <relation name="lawd:hasAttestation" active="LIT6624Maqabis" passive="BNFabb59"/>
```
Comment (eu-genia) [inline new/LIT6624Maqabis.xml:58]: I think we need the person record and the relation to the person

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6624Maqabis.xml (hunk 1 above)
Comment (thea-m) [reply new/LIT6624Maqabis.xml:58]: I can create him of course, but his identity remains quite unclear (I assume a desert father).

Comment (DenisNosnitsin1970) [reply new/LIT6624Maqabis.xml:58]: If he is the protagnist, do we need a personal record?

Comment (eu-genia) [reply new/LIT6624Maqabis.xml:58]: > I can create him of course, but his identity remains quite unclear (I assume a desert father).

Well we do know that he is venerated in the Synaxarion, this is justification enough

**Comment by thea-m** [comment]
https://github.com/BetaMasaheft/Documentation/issues/2095

File: new/LIT6624Maqabis.xml (hunk 1 above)
Comment (thea-m) [reply new/LIT6624Maqabis.xml:58]: https://github.com/BetaMasaheft/Persons/pull/556

**Comment by thea-m** [comment]
Can this be merged?

**Pull request merged**